src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -3090,6 +3267,57 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
         eval quick_visualization_preview((value) model.nodes.set_quick_preview(*value));
     }
 
+
+
+    // =============
+    // === Modes ===
```

Comment:
Please add info what are these "modes" here. Are these graph "display modes" ? I mean, this should be understandable while browsing the codebase and graph editor "mode" is not descriptive enough IMO.